baseStats.json

```diff
@@ -321,13 +321,12 @@
       "evolvesTo": 0,
       "evolvesFrom": "Kakuna",
       "quickMoves": [
-        "Bug Bite",
-        "Poison Jab"
+        201,
+        224
       ],
       "cinematicMoves": [
-        "Sludge Bomb",
-        "Aerial Ace",
-        "X Scissor"
+        90,
+        45
```

Comment:
Yep, seems like the parser missed a couple attacks somehow. Will fix it once im back at home. thanks for spotting ;)